### ISSUE TITLE
Remove IE-EDGE from starter template as not required now ( Flexbox only, IE9 Drop )

### ISSUE THREAD

**Issue by IamManchanda** [twbs/bootstrap#21476]
There is no need for the below code so please remove this from starter template 
`<meta http-equiv="X-UA-Compatible" content="IE=Edge">`

Source: http://stackoverflow.com/a/6771584 
( Same link that @mdo posted in his codeguide )

Here are the important quotes to consider from this comment 

> Depending upon what Microsoft browsers you support you may not need to continue using the X-UA-Compatible tag. If you need to support IE 9 or IE 8, then I would recommend using the tag. If you only support the latest browsers (IE 11 and/or Edge) then I would consider dropping this tag altogether. 

![ie-edge](https://cloud.githubusercontent.com/assets/4970624/21563363/a520744e-cea7-11e6-9876-a4acb904cac6.png)

**Comment by bardiharborow** [comment]
IE 10 will default to edge mode regardless of this directive if used with a HTML5 doctype, except in certain circumstances (intranet sites). In IE11, ["document modes are deprecated and should no longer be used"](https://msdn.microsoft.com/library/bg182625(v=vs.85).aspx).

**Comment by IamManchanda** [comment]
Yup @bardiharborow . This might be a bold move for some but this should be taken as we should remove any crutches behind which are of no use for the new, modern and flexibile web
